iiif_prep/iiif_prep/archivesspace.py

```diff
@@ -0,0 +1,38 @@
+from asnake.aspace import ASpace
+
+
+class NoResultsError(Exception):
+    pass
+
+
+class MultipleResultsError(Exception):
+    pass
+
+
+class ArchivesSpaceClient:
+    def __init__(self, baseurl, username, password):
+        self.client = ASpace(
+            baseurl=baseurl,
+            username=username,
+            password=password).client
+
+    def get_diary_refid(self, diary):
+        refid = None
+        search_url = "repositories/2/search?page=1&type[]=archival_object&q=digital_object:+{}.pdf".format(
+            diary)
```

Comment:
So the thing that was happening here was that passing params as a dict meant that `requests` was URL encoding them. So instead of getting the literal `:+` you were getting something else.